task-requests/style.css

```diff
@@ -85,11 +85,79 @@ body {
 .container__body {
   margin-top: 2.5rem;
 }
-.sort-filters {
+
+.filters__header {
+  display: none;
+  gap: 1rem;
+  align-items: center;
+  margin-bottom: 8px;
+}
+
+.filters__header > h2 {
+  margin: 0;
+  font-size: 1.25rem;
+}
+
+.filters__header > button {
+  background: none;
+  cursor: pointer;
+  padding: 3px 8px;
+  border: 1px solid var(--light-gray-color);
+  border-radius: 7px;
+}
+
+.filters__header > button:hover {
+  background-color: var(--red-color);
+  color: var(--white);
+}
```

Comment:
Using CSS variables for colors, font sizes, and spacing can help maintain a consistent style. we can define these at root level for easy global updates.

Reply:
most of the things are already using CSS variables, added variable for border-radius as well.